client/components/CandidateTests/CandidateTestPlanRun/index.jsx

```diff
@@ -0,0 +1,698 @@
+                                    true,
+                                    index === 0
+                                        ? 'changes-requested'
+                                        : 'feedback',
+                                    null,
+                                    true,
+                                    !differentAuthors ? data.me.username : null
+                                )}
+                            />
+                        );
+                    }
+                })}
+            </ul>
+        </div>
+    );
+
+    const results = (
+        <>
+            <h1 className="current-test-title">{currentTest.title}</h1>
+            <DisclosureComponent
+                componentId="test-instructions-and-results"
+                headingLevel="1"
+                title={[
+                    'Test Instructions',
+                    ...testPlanReports.map(
+                        testPlanReport =>
+                            `Test Results for ${testPlanReport.browser.name}`
+                    )
+                ]}
+                onClick={[
+                    () => setShowInstructions(!showInstructions),
+                    ...showBrowserClicks
+                ]}
+                expanded={[showInstructions, ...showBrowserBools]}
+                disclosureContainerView={[
+                    <InstructionsRenderer
+                        key={`instructions-${currentTest.id}`}
+                        at={testPlanReport.at}
+                        testResult={{
+                            scenarioResults: {},
+                            test: currentTest,
+                            completedAt: new Date()
+                        }}
+                        testPageUrl={testPlanReport.testPlanVersion.testPageUrl}
+                    />,
+                    ...testPlanReports.map(testPlanReport => {
+                        const testResult =
+                            testPlanReport.finalizedTestResults[
+                                currentTestIndex
+                            ];
+                        const { testsPassedCount } = getMetrics({ testResult });
+                        return (
+                            <>
+                                <h2 className="test-results-header">
+                                    Test Result:{' '}
+                                    {testsPassedCount ? 'PASS' : 'FAIL'}
+                                </h2>
+                                <TestPlanResultsTable
+                                    tableClassName="test-results-table"
+                                    key={`${testPlanReport.id} + ${testResult.id}`}
+                                    test={currentTest}
+                                    testResult={testResult}
+                                />
+                            </>
+                        );
+                    })
+                ]}
+                stacked="true"
```

Comment:
There's a warning in the console on this. The component is expecting the `stacked` prop to be a boolean. But this passes it as a string.

You could do `stacked={true}` or just `stacked`.